io/src/ply_io.cpp

```diff
@@ -213,6 +214,12 @@ namespace pcl
       }
       if (property_name == "alpha")
       {
+        if(pcl::getFieldIndex(*cloud_,"rgb") == -1)
```

Comment:
How about adding a `bool` return value to `amendProperty` indicating success/failure? That would avoid calling `getFieldIndex` twice